index.html

```diff
@@ -1,13 +1,13 @@
 <!DOCTYPE html>
 <html>
-<head style= "background-color: #e5ba57">
+<head class= "indexHead">
 	<meta charset="utf-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<title></title>
-	<link rel="stylesheet" href="">
+	<link rel="stylesheet" type="text/css" href="/home/mnixon/Sites/htmlPractice/style.css">
```

Comment:
Use a relative link here, so that if the projects location changes, your HTML document can still figure out where the CSS file is located.